File: OOAD/solid-app/src/ocp/lsp/violation/Rectangle.java
package ocp.lsp.violation;

public class Rectangle {

	protected int width;
	protected int height;
	
	Rectangle(int width,int height)
	{
		this.width=width;
		this.height=height;
	}

	public int getWidth() {
		return width;
	}

	public void setWidth(int width) {
		this.width = width;
	}

	public int getHeight() {
		return height;
	}

	public void setHeight(int height) {
		this.height = height;
	}
	
	public int calculateArea()
	{
		int area;
		area=height*width;
		return area;
	}
}

File: CSharpOOPS/rectangle-app/rectangle-app/rectangle/Rectangle.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace rectangle_app.rectangle
{
    class Rectangle
    {
        private int height;
        private int width;
        public Rectangle(int height,int width)
        {
            this.height = height;
            this.width = width;
        }

        public int calculateArea()
        {
            int area = width * height;
            return area;
        }
    }
}
